### ISSUE TITLE
Cycle Actions Scrollbar Annoyance

### ISSUE THREAD

**Issue by Jeff0S** [reaper-oss/sws#630]
_From [OuttrOMu...@gmail.com](https://code.google.com/u/116656722500254827865/) on February 10, 2014 02:55:23_

What steps will reproduce the problem? one more from me while i'm at it...
1.  create a cycle action with enough actions to introduce a vertical scrollbar in the cycle actions window
2.  try to drag the last action from the bottom of the list up one or two spots. What is the expected output? What do you see instead? it should be easy to reposition, but it's mostly impossible with any action below the bottom edge of the visible screen due to the scrollbar moving to the top as soon as any action is dragged.  

basically we're forced to view only the top of the list whenever rearranging items, so actions low in the list are very hard to move.

What version of SWS extension are you using? (Extensions -> About SWS...)

2.4.0#3 What is your Reaper version number and CPU architecture (e.g. Reaper 3.75 x64)? 3.59, x64 What operating system are you running?  (e.g. Windows XP/7, OSX 10.5.8 etc, 64bit?) win7 x64

_Original issue: http://code.google.com/p/sws-extension/issues/detail?id=630_
